packages/flutter_tools/lib/src/web/migrations/scrub_generated_plugin_registrant.dart

```diff
@@ -18,17 +18,16 @@ class ScrubGeneratedPluginRegistrant extends ProjectMigrator {
   final Logger _logger;
 
   @override
-  bool migrate() {
+  void migrate() {
     final File registrant = _project.libDirectory.childFile('generated_plugin_registrant.dart');
     final File gitignore = _project.parent.directory.childFile('.gitignore');
 
     if (!removeFile(registrant)) {
-      return false;
```

Comment:
This was the only migration that ever returned false.  We already have the error io file deletion fallbacks, and I'd argue failing to delete this file shouldn't be catastrophic and fail the web build.